cli/resource.go

```diff
@@ -24,7 +26,24 @@ func runResource(args *docopt.Args, client *controller.Client) error {
 	if args.Bool["add"] {
 		return runResourceAdd(args, client)
 	}
-	return fmt.Errorf("Top-level command not implemented.")
+
+	resources, err := client.AppResourceList(mustApp())
+	if err != nil {
+		return err
+	}
+
+	w := tabWriter()
+	defer w.Flush()
+
+	var provider *ct.Provider
+
+	listRec(w, "ID", "Provider ID", "Provider Name")
+	for _, j := range resources {
+		provider, err = client.GetProvider(j.ProviderID)
```

Comment:
> Should CLI commands halt on first error?

Until we add retry support, I think that makes sense.

> Perhaps tabular output to stdout, errors to stderr.

I'm pretty sure we already do this.